addons/popochiu/engine/objects/gui/components/dialog_text/dialog_portrait/dialog_portrait.gd

```diff
@@ -30,12 +35,16 @@ func _update_avatar(chr: PopochiuCharacter, _msg := '') -> void:
 		E.scale if E.settings.scale_gui else Vector2.ONE
 	)
 	
+	var flip_h := _LOOKING_LEFT_DIRS.has(chr._looking_dir)
```

Comment:
It looks like `_looking_dir` is being misused in other areas as well and should either be a getter/setter or a public variable.

Reply:
Yep, I guess that's very legacy code that nobody touched in a while. Nice spot!

Reply:
You're right @edmundito . We can make that property public (by removing the underscore at the beggining) or use getter/setter methods. I prefer the first option.

Changing that will also affect the `i_room.gd` and the `popochiu_room_data.gd` scripts. And this line of code, of course.